lib/components/molecules/StickyBar/StickyBar.style.js

```diff
@@ -0,0 +1,13 @@
+import { css } from 'styled-components';
+
+const StickyBarStyles = css`
+  display: flex;
+  flex-direction: ${props => (props.listDirection ? props.listDirection : 'column')};
+  top: ${props => (props.verticalAlign ? props.verticalAlign : `50%`)};
```

Comment:
What if there are too many elements in the Sticky bar and they gets cut off from bottom of the viewport? There should be some mechanism to handle such scenarios

Reply:
resolved by adding overflow: scroll and height: 100%